test/Benchmarks/src/Vector.enso

```diff
@@ -1,4 +1,5 @@
 from Standard.Base import all
+import Standard.Base
```

Comment:
Why is this needed?

Reply:
As the Benchmark is called `Vector` means you can access the `Standard.Base.Vector` as `Base.Vector`.

Reply:
Ah, okay. I was confused as there weren't any other changes.

But I assume before `from Standard.Base import all` was introducing `Base` into scope and this changed in one of the recent PRs and the benchmark was just not tested since then. So now it makes sense, thanks for the explanation.